internal/autodetect/main.go

```diff
@@ -83,3 +91,15 @@ func (a *autoDetect) OpenShiftRoutesAvailability() (openshift.RoutesAvailability
 
 	return openshift.RoutesNotAvailable, nil
 }
+
+func (a *autoDetect) RBACPermissions(ctx context.Context) (autoRbac.Availability, error) {
+	w, err := autoRbac.CheckRbacPermissions(ctx, a.reviewer)
```

Comment:
nit: can we make the naming convention consistent? I.e. RBAC vs Rbac (I think the all caps probs makes more sense)